### PR TITLE
feat: connection historic

### PR DESCRIPTION
**Pull request by Narkfr**

### CODE REVIEW

**Comment by Narkfr** [comment]
[(5) [SET-21] ETU j'ai accès à l'historique des connexions](https://trello.com/c/W4PxruTZ/630-5-set-21-etu-jai-acc%C3%A8s-%C3%A0-lhistorique-des-connexions)

File: src/components/SessionsView.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                        new Date(session.created_at),
+                        t('SessionsView.sync_date_format')
+                      )
+                    }
+                  </div>
+                  <div className={classNames(tableStyles['coz-table-cell'], tableStyles['set-table-os'])}>
+                    {session.os}
+                  </div>
+                  <div className={classNames(tableStyles['coz-table-cell'], tableStyles['set-table-browser'])}>
+                    {parser(session.user_agent).browser.name} {parser(session.user_agent).browser.major}
```
Comment (CPatchane) [inline src/components/SessionsView.jsx]: Maybe parsing the `session.user_agent` only once before the `render` function could be better, what do you think?

Comment (Narkfr) [reply src/components/SessionsView.jsx]: session is defined in the map function, I can't access it in the render. 
But I can make a sub component <SessionViewRow session={session} /> :+1:

File: src/components/SessionsView.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import React from 'react'
+import classNames from 'classnames'
+import parser from 'user-agent-parser'
+
+import Loading from './Loading'
+
+import { translate } from 'cozy-ui/react/helpers/i18n'
+
+const SessionsView = ({ t, f, isFetching, sessions, deleteOtherSessions }) => {
+  return (
```
Comment (CPatchane) [inline src/components/SessionsView.jsx]: You can put just above here

```
const user_agent = parser(session.user_agent)
```

can't you?

**Comment by CPatchane** [comment]
Thanks @Narkfr 🎉

**Pull request merged**